README.md

```diff
@@ -234,6 +234,24 @@ log4js.configure({
 
 ### Changelog
 
+### v4.2.x to > v4.3.x
+
+Attention: The following functionality works with nodejs v10.14.2 and higher. In previous node versions there was a problem not resolving async iteration on destroyed streams
+
+RPC stream enhancement: Ability to stop communication on demand when requested data received is sufficient
```

Comment:
Should we mention that this will work only when backpressure is true?

Reply:
Done